src/audio_playback/mod.rs

```diff
@@ -24,19 +28,22 @@ impl SimpleTemporaryPlayer {
 
         let params = synth.stream_params();
 
-        let soundfont: Arc<dyn SoundfontBase> = Arc::new(
-            SampleSoundfont::new("D:/Midis/Loud and Proud Remastered/Kaydax Presets/Loud and Proud Remastered (Realistic).sfz", params.clone()).unwrap(),
-        );
+        if !sfz_path.is_empty() {
+            let soundfont: Arc<dyn SoundfontBase> =
+                Arc::new(SampleSoundfont::new(sfz_path, params.clone()).unwrap());
```

Comment:
Technically this only gets called when you load a new file, rather than when you change the soundfont in the settings. 
In the future, I have ideas for how to make a dynamic state machine for keeping track of the exact sound output state, kinda like I did with the timer system.

Oh, also, unwrap bad, could crash here instead of ignoring invalid soundfonts.

You can leave this as-is for now, we will be rewriting a bunch of stuff here in the future, but yeah just something to keep in mind.

Reply:
It's like this so it doesn't crash if the user hasn't specified an SFZ file yet.
Also yes, I'll fix the unwrap.